src/rez/rex_bindings.py

```diff
@@ -173,6 +173,14 @@ def __init__(self, requirements):
     def _attr_error(self, attr):
         raise AttributeError("request does not exist: '%s'" % attr)
 
+    def get_range(self, name, default=None):
+        """Returns requirement version range object"""
+        req_str = self._data.get(name)
+        if req_str:
+            return Requirement(req_str).range
+        else:
+            return VersionRange(default)
```

Comment:
I think this is unexpected... much like get(), I think people would expect to get None if a default isn't specified